includes/overlords/contribs.php

```diff
@@ -290,8 +362,21 @@ public static function display_contribs($mode, $id, $branch = false, $sort = fal
 		$result = phpbb::$db->sql_query_limit($sql, $sort->limit, $sort->start);
 
 		$contrib_ids = $user_ids = array();
+
 		while ($row = phpbb::$db->sql_fetchrow($result))
 		{
+			if ($check_hidden_categories_on_all)
+			{
+				$contrib_categories = explode(',', $row['contrib_categories']);
+
+				if (sizeof($contrib_categories) == 1 && in_array($contrib_categories[0], $hidden_categories_ids))
```

Comment:
Please use `count` instead of `sizeof`and also `===`

Reply:
Why do you want to start using `count`? I can't find any references to `count` in all of Titania outside of dependencies. Everything that's currently there uses `sizeof` (probably because that was the standard for 3.0.x).

Reply:
Okay I asked @marc1706 - we're using `count` in phpBB 3.2 now. I'll use `count` here as well given that information, but I still think it's weird that we'd introduce the inconsistency into Titania.